### ISSUE TITLE
Accessibility : Tab Bar -Platform

### ISSUE THREAD

**Issue by Karthickc1994** [SAP/fundamental-ngx#8188]
Tab bar– Platform

1.1 Message Details
Raise Ticket	
Category	
Preprocessed By	
Description	

1.2 Application & Test Environment Details
Application Name:	ARIBA Controls- Fundamental Library for Angular
URL:	https://fundamental-ngx.netlify.app/#/platform/icon-tab-bar
Web browser:	Google Chrome Version 102.0.5005.63 (Official Build) (64-bit)
Operating System:	Microsoft Windows 10 x64
Screen Reader:	JAWS 2022.2204.20ILM

 
Issue 1

Priority:	High
Checkpoint:	ACC-264.1

Screenflow:
Within text type group.

Observed Behavior:
When access the tab bar ‘Item 1 (96)’ within the  text type tabs group,  the ‘Red color dot’ symbol present in the tab bar ‘Item 1 (96) is not read out. So the purpose of the UI is not read out.

Expected Behavior:
When accessing the tab bar, all the ‘Red color dot’ information and purpose of the UI  should read out to the user.

Screenshot:
![image](https://user-images.githubusercontent.com/105037808/172318099-54b23014-ae85-4b00-931d-a276f731df4b.png)

 
Issue 2
Priority:	High
Checkpoint:	ACC-264

Screen flow:
Within text type tabs.

Observed Behavior:
When accessing the list item within the more menu button incorrect type of UI read out.
Reading made as ‘Link instead of list’

Expected Behavior:
When accessing the list item within the more menu button type of UI list should read out clearly.
Screenshot:
![image](https://user-images.githubusercontent.com/105037808/172318171-5e79b930-1b30-4967-bac7-96fa1f8fcc83.png)
 
Case2:
![image](https://user-images.githubusercontent.com/105037808/172318197-84415fd3-e7e5-4dee-b75c-de03112e59f2.png)
 

Issue 3
Priority:	Medium
Checkpoint:	ACC-264.1

Screen flow:
Navigate to text type tabs.

Observed Behavior:
When accessing the tabs within the group’s additional information such as ‘Control+ Page down’ is read out.

Tester note: 
The reported issue persist all the tabs.

Expected Behavior:
When accessing the tabs additional information should be removed and only the relevant information should read out.

Screenshot:
![image](https://user-images.githubusercontent.com/105037808/172318332-9896cb79-6a98-441e-8891-24b42145b272.png)

 
Issue 4
Priority:	Medium
Checkpoint:	ACC-261.1

Screen flow:
Navigate to Icon type tabs.

Observed Behavior:
The focus indication provided for the tabs, does not meet the required ratio of 4.5:1.

Expected Behavior:
The focus indication provided for the tabs should satisfy the required ratio of 4.5:1.
Screenshot:
![image](https://user-images.githubusercontent.com/105037808/172318399-9a3b858e-6195-44a9-8d2e-7ee2adc26d8c.png)
 

Issue 5
Medium
Priority:	Medium
Checkpoint:	ACC-255.1

Screen flow:
Navigate to icon type tabs.

Observed Behavior:
The icon button within the tab’s does not have any alternative to describe the purpose of the UI.
Expected Behavior:
The icon button within the tabs should have visible clear alternative to describe the purpose of the UI.

Screenshot:
![image](https://user-images.githubusercontent.com/105037808/172318483-2011d2ab-69b2-41f4-9856-d2ac4febe617.png)

Thanks and Regards,
Karthikeyan.c
